src/main/resources/specifications.properties

```
@@ -0,0 +1,513 @@
+XEP-0408=Mediated Information eXchange (MIX): Co-existence with MUC
+XEP-0409=IM Routing-NG
+XEP-0410=MUC Self-Ping (Schr�dinger's Chat)
+XEP-0411=Bookmarks Conversion
+XEP-0412=XMPP Compliance Suites 2019
+XEP-0413=Order-By
+XEP-0414=Cryptographic Hash Function Recommendations for XMPP
+XEP-0415=XMPP Over RELOAD (XOR)
+XEP-0416=E2E Authentication in XMPP
+XEP-0417=E2E Authentication in XMPP: Certificate Issuance and Revocation
+XEP-0418=DNS Queries over XMPP (DoX)
+XEP-0419=Improving Baseline Security in XMPP
+XEP-0420=Stanza Content Encryption
+XEP-0421=Anonymous unique occupant identifiers for MUCs
+XEP-0422=Message Fastening
+XEP-0423=XMPP Compliance Suites 2020
+XEP-0424=Message Retraction
+XEP-0425=Moderated Message Retraction
+XEP-0426=Character counting in message bodies
+XEP-0427=MAM Fastening Collation
+XEP-0428=Fallback Indication
+XEP-0429=Special Interests Group End to End Encryption
+XEP-0430=Inbox
+XEP-0431=Full Text Search in MAM
+XEP-0432=Simple JSON Messaging
+XEP-0433=Extended Channel Search
+XEP-0434=Trust Messages (TM)
+XEP-0435=Reminders
+XEP-0436=MUC presence versioning
+XEP-0437=Room Activity Indicators
+XEP-0438=Best practices for password hashing and storage
+XEP-0439=Quick Response
+XEP-0440=SASL Channel-Binding Type Capability
+XEP-0441=Message Archive Management Preferences
+XEP-0442=Pubsub Message Archive Management
+XEP-0443=XMPP Compliance Suites 2021
+XEP-0444=Message Reactions
+XEP-0445=Pre-Authenticated In-Band Registration
+XEP-0446=File metadata element
+XEP-0447=Stateless file sharing
+XEP-0448=Encryption for stateless file sharing
+XEP-0449=Stickers
+XEP-0450=Automatic Trust Management (ATM)
+XEP-0451=Stanza Multiplexing
+XEP-0452=MUC Mention Notifications
+XEP-0453=DOAP usage in XMPP
+XEP-0454=OMEMO Media sharing
+XEP-0455=Service Outage Status
+XEP-0456=Content Rating Labels
+XEP-0457=Message Fancying
+XEP-0458=Community Code of Conduct
+XEP-0459=XMPP Compliance Suites 2022
+XEP-0460=Pubsub Caching Hints
+XEP-0461=Message Replies
+XEP-0462=PubSub Type Filtering
+XEP-0463=MUC Affiliations Versioning
+XEP-0464=Cookies
+XEP-0465=Pubsub Public Subscriptions
+XEP-0466=Ephemeral Messages
+XEP-0467=XMPP over QUIC
+XEP-0468=WebSocket S2S
+XEP-0469=Bookmark Pinning
+XEP-0470=Pubsub Attachments
+XEP-0471=Events
+XEP-0472=Pubsub Social Feed
+XEP-0473=OpenPGP for XMPP Pubsub
+XEP-0474=SASL SCRAM Downgrade Protection
+XEP-0475=Pubsub Signing
+XEP-0476=Pubsub Signing: OpenPGP Profile
+XEP-0477=Pubsub Targeted Encryption
+XEP-0478=Stream Limits Advertisement
+XEP-0479=XMPP Compliance Suites 2023
+XEP-0480=SASL Upgrade Tasks
+XEP-0481=Content Types in Messages
+XEP-0482=Call Invites
+XEP-0483=HTTP Online Meetings
+XEP-0484=Fast Authentication Streamlining Tokens
+XEP-0485=PubSub Server Information
+XEP-0486=MUC Avatars
+XEP-0487=Host Meta 2 - One Method To Rule Them All
+XEP-0488=MUC Token Invite
+XEP-0489=Reporting Account Affiliations
+XEP-0490=Message Displayed Synchronization"specs.put();
```

Typo? Should this be just
```suggestion
XEP-0490=Message Displayed Synchronization
```
?